pkg/amplitude/amplitude.go

```diff
@@ -44,11 +44,16 @@ func GetDeviceID() string {
 
 // GetEventOptions returns default options to be used when tracking an event.
 func GetEventOptions() (options ampli.EventOptions) {
+	osType := runtime.GOOS
+	if dockerutil.IsWSL2() {
+		osType = "wsl2"
```

Comment:
```suggestion
		osType += " (wsl2)"
```

What about just adding a suffix?